crates/control/src/behavior/dribble.rs

```diff
@@ -102,7 +105,7 @@ pub fn execute(
         kick_decisions
             .iter()
             .filter(|decision| {
-                !is_inside_any_obstacle(
+                is_inside_any_obstacle(
```

Comment:
This doesn't seem right to me. Why did you invert this?

Reply:
uh oh... good catch, i wanted to do expose all kick decision candidates (see https://github.com/HULKs/hulk/pull/171/commits/7426749b6fde8315ceb07e4cea636f04fde38083 ). Should be fixed now